### PR TITLE
added line

### PR DESCRIPTION
**Pull request by 3621566**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # cr
 pull request will be change
-change request
\ No newline at end of file
+
+change request
+
+new line added
```
Comment (iveter) [inline README.md:6]: please add today date

**Review by iveter — COMMENTED** [review]
read comments

File: README.md (hunk 1 above)
Comment (iveter) [inline README.md:6]: add date

**Review by iveter — CHANGES_REQUESTED** [review]

**Review by iveter — CHANGES_REQUESTED** [review]
add date

File: README.md (hunk 1 above)
Comment (3621566) [inline README.md:6]:
```suggestion
new line added 01/31/2023
```

**Review by 3621566 — COMMENTED** [review]
added

**Review by iveter — CHANGES_REQUESTED** [review]
please add author

**Review by 3621566 — COMMENTED** [review]
add author

**Review by iveter — APPROVED** [review]

**Pull request merged**